packages/itwinui-variables/src/themes/dark.scss

```diff
@@ -4,38 +4,73 @@
 @use '../utils/hex' as *;
 @use '../utils/get-hsl' as *;
 
-$base-colors: (
-  // Background colors - default
-  background-1: hsl(204 12% 40%),
-  background-2: hsl(202 12% 35%),
-  background-3: hsl(204 13% 31%),
-  background-4: hsl(200 14% 25%),
-  background-5: hsl(200 14% 21%),
-  background-1-hover: hsl(204 12% 45%),
-  //
-  // Background colors - statuses subtle
-  background-subtle-primary: hsl(204 14% 44%),
-  background-subtle-positive: hsl(192 9% 43%),
-  background-subtle-warning: hsl(201 6% 44%),
-  background-subtle-negative: hsl(216 7% 44%),
-  //
-  // Foreground colors - default
-  foreground-1: hsl(0 0% 100%),
-  foreground-2: hsl(210 9% 91%),
-  foreground-3: hsl(207 8% 79%),
-  foreground-4: hsl(206 8% 67%),
-  foreground-5: hsl(205 7% 55%),
-  //
-  // Foreground colors - statuses
-  foreground-primary: hsl(203 80% 80%),
-  foreground-positive: hsl(96 45% 78%),
-  foreground-warning: hsl(34 87% 82%),
-  foreground-negative: hsl(0 69% 80%),
-  foreground-primary-hover: hsl(203 80% 90%),
-  foreground-positive-hover: hsl(96 44% 88%),
-  foreground-warning-hover: hsl(34 85% 92%),
-  foreground-negative-hover: hsl(0 69% 90%)
-);
+// #region SASS variables
+$bg-hue: 203;
+$bg-saturation: 12%; // More saturated than high contrast theme.
+$bg-lightness-gap: 3.25%;
+$fg-lightness-gap: 7%;
+// #endregion
+
+// #region Background colors
+// Darkest @ top, lightest @ bottom
+$bg-0: hsl($bg-hue $bg-saturation 0%);
+$bg-1: hsl($bg-hue $bg-saturation ($bg-lightness-gap));
+$bg-2: hsl($bg-hue $bg-saturation ($bg-lightness-gap * 2));
+$bg-3: hsl($bg-hue $bg-saturation ($bg-lightness-gap * 3));
+$bg-4: hsl($bg-hue $bg-saturation ($bg-lightness-gap * 4));
+$bg-5: hsl($bg-hue $bg-saturation ($bg-lightness-gap * 5));
+$bg-6: hsl($bg-hue $bg-saturation ($bg-lightness-gap * 6));
+$bg-7: hsl($bg-hue $bg-saturation ($bg-lightness-gap * 7));
+$bg-8: hsl($bg-hue $bg-saturation ($bg-lightness-gap * 7.5));
```

Comment:
Same as other comment - have you thought about moving this color to be outside of the series? Maybe call it `$bg-7b` instead of `&bg-8`?